### ISSUE TITLE
[PT] Update lesson: Enum 

### ISSUE THREAD

**Issue by thiamsantos** [elixirschool/elixirschool#2396]
## Motivation

The portuguese translation for the lesson [Enum](https://elixirschool.com/en/lessons/basics/enum/) needs to be updated.
We need to update the version of the lesson from `1.5.0` to `1.6.1`.

Related #2382

## Proposed Solution

Translate the updates of the lesson.

You can run `git diff` to see all differences between the versions of the lesson:

```sh
git diff 482398d61a27cd3dd26d39a5e68c9222becf9ced HEAD en/lessons/basics/enum.md
```

## Tips

- Check the [contributing guidelines](https://github.com/elixirschool/elixirschool/blob/master/CONTRIBUTING.md) before starting.
- Feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/#tag-your-work-in-progress) as soon as you start working on the translation. In order to prevent we prevent having multiple people working on the translation and also allow the reviewers to give early feedback.
- In case of any doubt feel free to comment in this issue.

**Comment by alessandraburckhalter** [comment]
Hi there! Can I work on this?

**Comment by vbrazo** [comment]
@alessandraburckhalter yes, please :)

**Comment by thiamsantos** [comment]
Closed by #2409
